tests/components/test_auto_fill.py

```diff
@@ -18,9 +18,23 @@ def test_auto_fill_happy_path(kwik_e_mart_app, kwik_e_mart_app_path, qa_kwik_e_m
     assert_target_dialogue_state(convo, None)
 
 
+@pytest.mark.conversation
+def test_auto_fill_happy_path_validation(kwik_e_mart_app, kwik_e_mart_app_path, qa_kwik_e_mart):
+    """Tests a happy path for the app with gazetter validation."""
+    convo = Conversation(
+        app=kwik_e_mart_app, app_path=kwik_e_mart_app_path, force_sync=True
```

Comment:
I think you don't need the rest of the params
```suggestion
        app=kwik_e_mart_app)
```